src/js/controller/PaletteController.js

```diff
@@ -106,11 +110,21 @@
     } else {
       colorPicker.spectrum('set', color);
     }
-    this.setTitleOnPicker_(color, colorPicker);
   };
 
-  ns.PaletteController.prototype.setTitleOnPicker_ = function (title, colorPicker) {
-    var spectrumInputSelector = '.sp-replacer';
-    colorPicker.next(spectrumInputSelector).attr('title', title);
+  /**
+   * @private
+   */
+  ns.PaletteController.prototype.showValueDisplayer_ = function (evt) {
```

Comment:
Yes exactly. The tooltip could be on two lines, with the second line displaying the current color.

Sorry about the rushed comments/review. I had to leave in a hurry 5mn after starting the review.